crypto/cryptlib.c

```diff
@@ -120,6 +124,23 @@ void OPENSSL_cpuid_setup(void)
 # endif
 
 # if defined(_WIN32_WINNT) && _WIN32_WINNT>=0x0333
+#  ifdef OPENSSL_SYS_WIN_CORE
+TRACELOGGING_DEFINE_PROVIDER(
+    g_hProvider,
+    "OpenSSL",
+    /*
+     * Product GUID generated by MS Tool (randomly), needed to send trace logs
+     * {5ee02bf4-33cf-53cf-ddc9-f18679e677dd} 
+     */
+    (0x5ee02bf4, 0x33cf, 0x53cf, 0xdd, 0xc9, 0xf1, 0x86, 0x79, 0xe6, 0x77, 0xdd)
+    );
```

Comment:
Well, suggestion was rather to align closing parenthesis with TRACELOGGING_DEFINE_PROVIDER, but all right...

Reply:
I wonder what to do if after alignment guid line may be too long,
Should I Split it? what is max line length in chars

Reply:
80

BTW, have you read our coding style document? 
https://www.openssl.org/policies/codingstyle.html